downstream/modules/platform/proc-settings-user-preferences.adoc

```diff
@@ -0,0 +1,50 @@
+[id="proc-settings-user-preferences"]
+
+= User preferences
+
+//To be added to Donna's AAP/UI document for 2.5 
+
+You can use the *User preferences* page to customize your platform experience. Use this menu to control theming, layout options and formatting.
+
+[NOTE]
+====    
+User preferences are stored locally in your browser. This means that they are unique to that user and that machine.
```

Comment:
Unique to your username and your machine?
Because you're addressing the user as "you" in the first half of the sentence.